### PR TITLE
[build] switch from travis to GH actions

### PR DESCRIPTION
**Pull request by pawelmhm**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+recursive-include scrapyd *.py
 recursive-include docs *
+recursive-include bin *
 prune docs/_build
 recursive-include extras *
+recursive-exclude debian *
+exclude .bumpversion.cfg
+exclude .coveragerc
+exclude *.buildbot
+include tox.ini
```
Comment (jpmckinney) [inline MANIFEST.in:16]: Looks like tox.ini is deleted?

File: scrapyd/tests/test_utils.py [hunk 2]
```diff
@@ -98,6 +99,10 @@ def test_get_spider_list(self):
         self.assertEqual(sorted(spiders), ['spider1', 'spider2'])
 
     def test_get_spider_list_unicode(self):
+        # TODO fix this test on windows
+        # TODO switch to pytest from trial
+        if os.name == 'nt':
+            raise SkipTest
```
Comment (jpmckinney) [inline scrapyd/tests/test_utils.py:105]: I assume the PR is still draft?

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            python-version: pypy-3.7
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install system libraries
+      if: contains(matrix.os, 'ubuntu') && contains(matrix.python-version, 'pypy')
+      run: |
+        sudo apt-get update
+        sudo apt-get install libxml2-dev libxslt-dev
```
Comment (jpmckinney) [inline .github/workflows/tests.yml:26]: Which packages require these?

File: scrapyd/tests/test_utils.py (hunk 2 above)
Comment (jpmckinney) [reply scrapyd/tests/test_utils.py:105]: If the plan is to leave these TODOs, then GitHub issues should be created for each of them.

File: .github/workflows/tests.yml (hunk 3 above)
Comment (pawelmhm) [reply .github/workflows/tests.yml:26]: Scrapy. I followed your example here: https://github.com/scrapy/scrapyd-client/commit/279330eae3e05646f7905aeb2edab4777c136b46 but actually let's try without this and see what's up.

**Comment by pawelmhm** [comment]
thanks for review and feedback @jpmckinney I included your comments.

Sorry for multiple commits, I'll squash this into one commit on merge.

File: scrapyd/tests/test_utils.py (hunk 2 above)
Comment (pawelmhm) [reply scrapyd/tests/test_utils.py]: fixed here https://github.com/scrapy/scrapyd/pull/412/commits/50e47f26a7682aaefc81737d18b0b332c7d52bfb

File: MANIFEST.in (hunk 1 above)
Comment (pawelmhm) [reply MANIFEST.in]: good catch thanks fixed https://github.com/scrapy/scrapyd/pull/412/commits/50e47f26a7682aaefc81737d18b0b332c7d52bfb

File: .github/workflows/tests.yml (hunk 3 above)
Comment (pawelmhm) [reply .github/workflows/tests.yml]: looks like it works fine without this step, so I removed it https://github.com/scrapy/scrapyd/pull/412/commits/61bbff591d746679f817e463eaebdce92a27888d

**Comment by jpmckinney** [comment]
🎉 

I think the `SkipTest` import is no longer needed, though.

**Comment by pawelmhm** [comment]
good point, I fixed that here: https://github.com/scrapy/scrapyd/pull/413/files

**Pull request merged**